src/__tests__/usage-test.ts

```diff
@@ -11,7 +9,7 @@ import { graphqlHTTP } from '../index';
 describe('Useful errors when incorrectly used', () => {
   it('requires an option factory function', () => {
     expect(() => {
-      // $DisableFlowOnNegativeTest
+      // @ts-expect-error should return runtime error
```

Comment:
These messages don't really bring any value so just change eslint config to not require a message.